docs/contributing.md

```diff
@@ -2,81 +2,85 @@
-  - Switch the assignee through meeting or request. 
-
-### 1.3. Processing assigned issues during the Sprints
-> Validator Team's ultimate existence reasons are
-- Maximizing uptime
-- Following up managed schedule per protocols (binary updates, new spork, epoch, vote, etc)
-- Contributing as validator to boost protocol if there's any further improvements are available.
-
-1. At least 1 to 2 issues has to be addressed within a Sprint(two weeks).
-2. Assignee has to finish assigned or selected issue as much as possible, leave a comment if it can't be finished within sprint.
-3. As Validator team's main task is node operation and therefore leave issue handling to each member's own will. 
-   Close the issue, re-register and change to other assignee if it is difficult to proceed during the 2 sprints(4 weeks).
-
-### 1.4. PR policy
-> This is rules for PR for VATZ project. 
+- `CI/CD Implementation`:
+   Please use this template for the purpose of setting a CI/CD pipeline for Vatz (i.e, labeling, and discussion for new policies.)
+
+### step 3. Fill out the info
+
+Please, fill out all the following info as much as possible when you register the issue.
+- Assignee
+- Labels
+- Projects
+- Milestone
+
+<img width="335" alt="image" src="https://user-images.githubusercontent.com/6308023/164614699-2ddeea3f-b0c6-45db-be28-7193afb613cc.png">
+
+
+## 2. Assign assignee
+**Voluntary**
+
+Anyone can assign themselves if they want to do certain tasks.   
+
+**Mandatory**
+
+The code owner(@xellos00) sets an assignee if there's an appropriate assignee available in the team for certain tasks. We may have a discussion prior to setting an assignee if needed.
+
+  Please, let the code owner(@xellos00) know if there are difficulties with the assigned task.
+  - Go over with everyone on Vatz bi-weekly meeting
+  - Switch the assignee through a meeting or request.
+
+## 3. PR policy
+> These are rules for PR for VATZ project. 
 
 - PR - First Review approved / First Merge.
-- Delete Branch that has been merged.
-- Person who raises the PR must track their PR status until their PR is closed.
+- You must delete the branch that has been merged.
+- If you raise the PR, you must track their PR status until their PR is closed.
     - Anyone who comments on PR while reviewing the process has an obligation to resolve/close their comment when the assignee has fixed a comment or suggestion.
-- Must include one of keywords in `close` or `related` as below. 
+- You must include one of the keywords in `close` or `related` as below. 
    - Put `close` keyword when you would like to close an issue. 
-   - Put `related` keyword when you would like to comment relate to an issue. 
+   - Put `related` keyword when you would like to comment related to an issue. 
   
-## 2. External Process (For those who want to contribute to open source)
-(TBD)
\ No newline at end of file
+
+  ---
+
+
+## DSRV Validator Team's development process
+(This part is used only for the internal processes)
```

Comment:
Can you update this to This section is a record for DSRV's internal development process.

Reply:
updated!